src/main/java/com/dougnoel/sentinel/steps/APISteps.java

```diff
@@ -189,7 +190,17 @@ public void verifyResponseContains(String assertion, String matchType, String te
 	 */
 	@When("^I add an? (.*?) header with the value (.*?)$")
 	public void addHeader(String name, String value) {
-		APIManager.addHeader(name, value);
+		APIManager.addHeader(name, SentinelStringUtils.replaceStoredVariables(value));
 	}
 
+	/**
+	 * Sets the body of the active API call to the string passed.
+	 *
+	 * @param body String the json to be passed as the body of the request.
+	 */
+	@Given("I set the request body with variables to")
```

Comment:
I don't know what this step means. The steps need to read like manual testing steps. If this test were to fail and I were to have to manually reproduce this, what would the step look like to reproduce what's happening here? There are also no usage examples in the Javadoc above. If this is an internal helper method, there should be no Given decorator, and this method should be private.